R/urls.R

```diff
@@ -0,0 +1,6 @@
+#' @title Base Urls
+BASE_URLS <- c(
+"staging" = "https://pasta-s.lternet.edu",
+"development" = "https://pasta-d.lternet.edu",
+"production" = "https://pasta.lternet.edu"
```

Comment:
url missing "/" in the end 
```suggestion
"staging" = "https://pasta-s.lternet.edu/",
"development" = "https://pasta-d.lternet.edu/",
"production" = "https://pasta.lternet.edu/"
```